src/dvclive/lightning.py

```diff
@@ -50,35 +50,19 @@ def _should_call_next_step():
 
 
 class DVCLiveLogger(Logger):
-    def __init__(  # noqa: PLR0913
+    def __init__(
         self,
         run_name: Optional[str] = "dvclive_run",
         prefix="",
         log_model: Union[str, bool] = False,
         experiment=None,
-        dir: Optional[str] = None,  # noqa: A002
-        resume: bool = False,
-        report: Optional[str] = None,
-        save_dvc_exp: bool = False,
-        dvcyaml: bool = True,
-        cache_images: bool = False,
+        **kwargs,
```

Comment:
Explicit kwargs initially came from https://github.com/iterative/dvclive/issues/178. This is no longer an issue in newer versions of lightning, and there are CLI tests already in lightning that I added below.